src/components/notifications/AppExplorer/AppExplorer.scss

```diff
@@ -87,7 +87,7 @@
 
   &__apps {
     width: 100%;
-    max-width: 700px;
+    max-width: 768px;
```

Comment:
Unlike the `md` screen breakpoint, this component is intended to be `700px` by design. So let's rever to `700px`